h/services/bulk_api/lms_stats.py

```diff
@@ -17,65 +17,83 @@ class AssignmentStats:
     last_activity: datetime
 
 
+@dataclass
+class CourseStats:
+    assignment_id: str
+    annotations: int
+    replies: int
+    last_activity: datetime
```

Comment:
According to the props, this class seems to represent the stats for an individual assignment. Should it be called `AssignmentStats` then?

I suppose you named it `CourseStats`, because the `"api.bulk.stats.course"` endpoint returns a list of these, but I think it might be confusing long term.